app/models/spree/image.rb

```diff
@@ -30,7 +30,7 @@ class Image < Asset
 
     # used by admin products autocomplete
     def mini_url
-      attachment.url(:mini, false)
+      url_for(attachment.variant(resize: "48x48#"))
```

Comment:
I think that we should define this size in one place. Maybe we can add methods to the Spree::Image class that call `attachment.variant` with our standardised sizes?